cognite/client/_api/functions.py

```diff
@@ -63,11 +63,11 @@ def _get_function_internal_id(cognite_client: CogniteClient, identifier: Identif
     raise ValueError(f'Function with external ID "{primitive}" is not found')
 
 
-def _get_function_identifier(function_id: int | None, function_external_id: str | None) -> Identifier:
-    identifier = IdentifierSequence.load(function_id, function_external_id, id_name="function")
+def _get_function_identifier(function_id: int | None) -> Identifier:
+    identifier = IdentifierSequence.load(function_id, id_name="function")
     if identifier.is_singleton():
         return identifier[0]
-    raise AssertionError("Exactly one of function_id and function_external_id must be specified")
+    raise AssertionError("Argument function_id must be specified")
```

Comment:
Functions still can be referenced with external id, so these changes must be reverted. It is only attaching a schedule that requires "id only".

Reply:
Ah, my misunderstanding. I will fix these later today.